Compile issue with gcc/g++ 13

When compiling, I've encountered this error:

```
kaldi_native_io.old/build  🍣 master via △ v3.26.4 via 🅒 k2 
🕙 10:44:50 ➜ make -j 5  
[  7%] Building CXX object kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/io-funcs.cc.o
[ 11%] Building CXX object kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/compressed-matrix.cc.o
[ 11%] Building CXX object kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/compressed-matrix.cc.o
[ 11%] Building CXX object kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/kaldi-holder.cc.o
[ 11%] Building CXX object kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/io-funcs.cc.o
In file included from /home/chicky/workspace/github/kaldi_native_io.old/kaldi_native_io/csrc/io-funcs-inl.h:16,
                 from /home/chicky/workspace/github/kaldi_native_io.old/kaldi_native_io/csrc/io-funcs.h:14,
                 from /home/chicky/workspace/github/kaldi_native_io.old/kaldi_native_io/csrc/io-funcs.cc:8:
/home/chicky/workspace/github/kaldi_native_io.old/kaldi_native_io/csrc/log.h:20:55: error: ‘uint32_t’ has not been declared
   20 |   Logger(const char *filename, const char *func_name, uint32_t line_num,
...
...
...
make[2]: *** [kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/build.make:104: kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/kaldi-holder.cc.o] Error 1
make[2]: *** [kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/build.make:76: kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/compressed-matrix.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:367: kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/build.make:76: kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/compressed-matrix.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:341: kaldi_native_io/csrc/CMakeFiles/kaldi_native_io_core_static.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

```
this happened due to this changes: https://gcc.gnu.org/gcc-13/porting_to.html#:~:text=Header%20dependency%20changes

--- a/kaldi_native_io/csrc/log.h
+++ b/kaldi_native_io/csrc/log.h
@@ -4,6 +4,7 @@
 
 #ifndef KALDI_NATIVE_IO_CSRC_LOG_H_
 #define KALDI_NATIVE_IO_CSRC_LOG_H_
+#include <cstdint>
 #include <iostream>
 #include <sstream>
 

--- a/kaldi_native_io/python/csrc/parse-options.h
+++ b/kaldi_native_io/python/csrc/parse-options.h
@@ -7,6 +7,7 @@
 #ifndef KALDI_NATIVE_IO_PYTHON_CSRC_PARSE_OPTIONS_H_
 #define KALDI_NATIVE_IO_PYTHON_CSRC_PARSE_OPTIONS_H_
 
+#include <cstdint>
 #include <sstream>
 #include <string>
 #include <unordered_map>

